_translations/en.yml

```diff
@@ -913,7 +913,7 @@ en:
     volatile: "Bitcoin price is volatile"
     volatiletxt: "The price of a bitcoin can unpredictably increase or decrease over a short period of time due to its young economy, novel nature, and sometimes illiquid markets. Consequently, keeping your savings with Bitcoin is not recommended at this point. Bitcoin should be seen like a high risk asset, and you should never store money that you cannot afford to lose with Bitcoin. If you receive payments with Bitcoin, many service providers can convert them to your local currency."
     irreversible: "Bitcoin payments are irreversible"
-    irreversibletxt: "Any transaction issued with Bitcoin cannot be reversed, they can only be refunded by the person receiving the funds. That means you should take care to do business with people and organizations you know and trust, or who have an established reputation. For their part, businesses need to keep control of the payment requests they are displaying to their customers. Bitcoin can detect typos and usually won't let you send money to an invalid address by mistake. Additional services might exist in the future to provide more choice and protection for the consumer."
+    irreversibletxt: "Any Bitcoin transaction, once accepted by the network, cannot be reversed, they can only be refunded by the person receiving the funds. That means you should take care to do business with people and organizations you know and trust, or who have an established reputation. For their part, businesses need to keep control of the payment requests they are displaying to their customers. Bitcoin can detect typos and usually won't let you send money to an invalid address by mistake. Additional services might exist in the future to provide more choice and protection for the consumer."
```

Comment:
Not your fault, but it looks like the previous string had some issues between plural and singular form. Let's revise your string slightly, to:

>A Bitcoin transaction, once accepted by the network, cannot be reversed. It can only be refunded by the recipient of the funds. This means you should take care to do business with parties you know and trust, or those who have an established reputation. For their part, businesses and individuals need to keep control of the wallets associated with the payment requests they are displaying. Bitcoin can detect typos and usually won't let you send money to an invalid address by mistake, but it's important to double-check. Additional services may also exist in the future to provide more choices and protections to users.
```